doc/ref_model/chapters/chapter09.md

```diff
@@ -243,3 +244,82 @@ auto.cicd.014 | The CD pipeline must be able to automatically promote software f
 <a name="9.5.3.2"></a>
 #### 9.5.3.2. Tenant Networking Automation
 
+
+<a name="9.6"></a>
+#### 9.6. Telemetry and Observability
+
+Operating complex distributed systems, such as Telco network, is a demanding and challenging task, being increased as the production excellence requirement grows. There are multiple reasons why it is so, but they originate in the nature of the system concept. To reach the ability of providing Telco services, a complex system is divided into multiple blocks with functional differences, called network functions. Internal communication between diversity of network functions of a distributed system is based on a message exchange. To formalize it, clearly defined interfaces are introduced, and protocols are being designed. Even though the architecture of Telco network is systematically formalized on the worldwide level, heterogeneity of services, functions, interfaces, and protocols cannot be avoided. By adding the multi-vendor approach in implementation of Telco networks, the outcome is indeed a system with remarkably high level of complexity which requires significant efforts for operating it.
+
+To ensure proper support and flawless work in the large ecosystem of end user services, a formalized approach directed towards high reliability and scalability of systems is required. The discipline which applies well known practices of software engineering to operations is called Site Reliability Engineering. It was conceived at Google, as a means to overcome missing limitations of the common DevOps approach.
+
+Common supporting system (OSS – Operation Support System, BSS – Business Support System) requirements are redefined, driven by introduction of new technologies in computing infrastructure; modern data centres with abstraction of resources – known as virtualization and cloud computing. This brings many advantages – such as easy scaling, error recovery, reaching a high level of operational autonomy etc., but also many new challenges in the Telecom network management space. Those novel challenges are mostly directed towards the dynamical nature of the system, orientation towards microservices instead of a silo approach, and huge amounts of data which has to be processed in order to understand the internal status of the system. Hence the need of improved ways to monitor systems - observability.
```

Comment:
Some thing is missing from this phrase, "modern data centres with abstraction of resources – known as virtualization and cloud computing"

Reply:
@pgoyal01 - This is fixed